### PR TITLE
Intro to build servers tali

### PR DESCRIPTION
**Pull request by TeganAli**
# About

This is a post about...

# Release

Is this post for a specific release? If so which one?

# Blog image idea

The design team will work on a design for the blog image. Add any ideas you have about the graphic image here.

# Pre-requisites

- [ ] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: An introduction to build servers
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:2]:
```suggestion
title: An introduction to build servers and continuous integration
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 2]
```diff
@@ -13,54 +13,55 @@ tags:
...
   - Jenkins
   - GitHub Actions
-  - Testing
 ---
 
-Octopus can receive packages by whatever means you want, whether you upload them direct to our in-built package store or import them via a feed. As believers in Continuous Integration/Continuous Delivery (CI/CD), though, we think they should come from a build server (also known as a 'CI platform').
+Octopus can receive packages whether you upload them to our in-built package store or import them via a feed. As believers in Continuous Integration and Continuous Delivery (CI/CD), though, we think packages should come from a build server, also known as a Continuous Integration (CI) platform.
 
-Given they can automate everything that happens between code commits and deployments, build servers are vital to CI as a concept. Let’s dig into exactly why we rate them so highly.
+Given build servers can automate everything that happens between code commits and deployments, they're vital to CI as a concept. Let’s dig into why we rate them so highly.
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:20]: Let's add a bit more context here. The piece jumps straight into what octopus can consume, which is important, but as we're targeting non octopus users, we can leave that bit for later :)  For instance:

```suggestion
When you're developing and deploying software, one of the first things you need to figure out is how to go from the code you and your team write on your laptops to deploying your working application into a production environment where end users can interact with your software. Most development teams understand the importance of version control to coordinate code commits and build servers to compile and package their software, but continuous integration is such a big topic, we've decided to spend some time over the next few months going into detail with continuous integration and two of the most popular build servers, Jenkins Pipelines and GitHub Actions.
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 3]
```diff
@@ -13,54 +13,55 @@ tags:
...
 
 ## Why build servers are important
 
 Build servers have 3 main purposes:
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:24]: I don't think we have this in the style guide, but I've always followed the Chicago manual of style for spelling out numbers below 100:

```suggestion
Build servers have three main purposes:
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 4]
```diff
@@ -13,54 +13,55 @@ tags:
...
 
-- compiling committed code from your repository many times a day
-- running thorough, automatic tests to validate code
-- creating deployable packages and handing off to your deployment tool.
+- Compiling committed code from your repository many times a day
+- Running thorough, automatic tests to validate code
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:27]: I've removed thorough as that feels fairly subjective:

```suggestion
- Running automatic tests to validate code
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 5]
```diff
@@ -13,54 +13,55 @@ tags:
...
+- Creating deployable packages and handing off to your deployment tool
 
-Life without a build server means time bogged down with manual processes and the needless time constraints caused by them. For example, without a build server:
+Without a build server you're bogged down with manual processes and the needless time constraints caused by them. For example, without a build server:
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:30]: Bogged is funny word :) feels very English and might be a struggle for ESL peeps. I also added complicated to the sentence as simplifying complex deployments is one of Octopus strongest points:

```suggestion
Without a build server you're slowed down by complicated, manual processes and the needless time constraints they introduce. For example, without a build server:
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 6]
```diff
@@ -13,54 +13,55 @@ tags:
...
 
 - Your team will likely need to commit code before a daily deadline or during change windows.
 - Once that deadline passes, no one can commit again until someone manually creates and tests a build.
 - If there are problems with code, the deadlines and manual processes delay the fixes further.
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:34]: Consistent punctuation:

```suggestion
- Your team will likely need to commit code before a daily deadline or during change windows
- After that deadline passes, no one can commit again until someone manually creates and tests a build
- If there are problems with code, the deadlines and manual processes further delay the fixes
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 7]
```diff
@@ -13,54 +13,55 @@ tags:
...
 
-As you can see, in this scenario the team battles hurdles that don't need to exist because of automation. A build server will repeatedly do all this for you throughout the day, and without those human-caused delays.
+In this scenario, the team battles unnecessary hurdles. A build server, however, repeatedly does all this for you throughout the day, and without those human-caused delays.
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:36]:
```suggestion
Without a build server, the team battles unnecessary hurdles that could be eliminated with automation. A build server, will repeatedly these tasks for you throughout the day, and without those human-caused delays.
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 8]
```diff
@@ -13,54 +13,55 @@ tags:
...
 
-But CI doesn’t just mean less time spent on manual tasks or the death of arbitrary deadlines, either. By automatically taking these steps many times a day, you’ll fix problems sooner and your results will become far more predictable. This, ultimately, helps you deploy through your pipeline with more confidence.
+But CI doesn’t just mean less time spent on manual tasks or the death of arbitrary deadlines, either. By automatically taking these steps many times a day, you fix problems sooner and your results become more predictable. Build servers ultimately help you deploy through your pipeline with more confidence.
 
-We’re taking deep dives into 2 CI platforms in the coming weeks, looking at how [Jenkins](https://www.jenkins.io/) or [GitHub Actions](https://github.com/features/actions) could help with your processes, plus how they complement Octopus.
+We’re taking deep dives into 2 CI platforms in the coming weeks, looking at how [Jenkins](https://www.jenkins.io/) or [GitHub Actions](https://github.com/features/actions) can help with your processes, plus how they complement Octopus.
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:40]: Slight switch of focus:
```suggestion
We’re taking deep dives into two CI platforms in the coming weeks, looking at how [Jenkins](https://www.jenkins.io/) or [GitHub Actions](https://github.com/features/actions) can help with your processes, and how Octopus can help you get more out of your CI/CD pipeline.
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 9]
```diff
@@ -13,54 +13,55 @@ tags:
...
 
-## A little about Jenkins
+## About Jenkins
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:42]: From an SEO perspective, I understand what is X questions in titles really help.
```suggestion
## What is Jenkins
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 10]
```diff
@@ -13,54 +13,55 @@ tags:
...
 [Jenkins](https://www.jenkins.io/) is the most popular CI platform on the market. Open-source and free to use, you can run Jenkins standalone on most operating systems to automate the building and testing of your code.
 
-One of its biggest benefits, however, is its flexibility. For one, it’s a scalable platform, meaning you can expand its capabilities as your team or project needs grow. And thanks in no small part to its huge community, [there over 1800 plugins](https://plugins.jenkins.io/), making it easy to integrate with countless industry tools.
+One of Jenkins biggest benefits is its flexibility. It's a scalable platform, meaning you can expand its capabilities as your team or project needs grow. And thanks to its huge community, [there over 1800 plugins](https://plugins.jenkins.io/), making it easy to integrate with countless industry tools.
  
-That means it’s not only flexible enough to cover your CI needs, but you can also tailor it for other automation purposes too.
+That means Jenkins is flexible enough to cover your CI needs and you can tailor it for other automation purposes too.
 
-## A little about GitHub Actions
+## About GitHub Actions
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:50]:
```suggestion
## What is GitHub Actions
```

File: blog/2022-01/an-introduction-to-build-servers/index.md (hunk 2 above)
Comment (TeganAli) [reply blog/2022-01/an-introduction-to-build-servers/index.md:20]: @wordlee I believe we're talking about Jenkins, not just [Jenkins Pipeline](https://www.jenkins.io/doc/book/pipeline/) for this series. I asked about it in the channel (#content-initiative-2021-q4) so you can see the discussion there.

Comment (TeganAli) [reply blog/2022-01/an-introduction-to-build-servers/index.md:20]: @wordlee can I also suggest the following small edits to your proposed content?

When you're developing and deploying software, one of the first things to figure out is how to take your code and deploy your working application to a production environment where people can interact with your software.

Most development teams understand the importance of version control to coordinate code commits, and build servers to compile and package their software, but Continuous Integration (CI) is a big topic. Over the next few months, we’re going into detail about Continuous Integration and how two of the most popular build servers, Jenkins and GitHub Actions, can help with your CI processes.

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 11]
```diff
@@ -10,57 +10,59 @@ bannerImageAlt: 125 characters max, describes image to people unable to see it.
 isFeatured: false
 tags:
   - DevOps
+  - CI series
```
Comment (TeganAli) [inline blog/2022-01/an-introduction-to-build-servers/index.md:13]:
```suggestion
  - CI Series
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 12]
```diff
@@ -10,57 +10,59 @@ bannerImageAlt: 125 characters max, describes image to people unable to see it.
...
   - DevOps
+  - CI Series
   - Continuous Integration
   - Jenkins
   - GitHub Actions
-  - Testing
 ---
 
-Octopus can receive packages by whatever means you want, whether you upload them direct to our in-built package store or import them via a feed. As believers in Continuous Integration/Continuous Delivery (CI/CD), though, we think they should come from a build server (also known as a 'CI platform').
+Octopus can receive packages whether you upload them to our in-built package store or import them via a feed. As believers in Continuous Integration and Continuous Delivery (CI/CD), though, we think packages should come from a build server, also known as a Continuous Integration (CI) platform.
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:19]:
```suggestion
When you're developing and deploying software, one of the first things to figure out is how to take your code and deploy your working application to a production environment where people can interact with your software.

Most development teams understand the importance of version control to coordinate code commits, and build servers to compile and package their software, but Continuous Integration (CI) is a big topic. Over the next few months, we’re going into detail about Continuous Integration and how two of the most popular build servers, Jenkins and GitHub Actions, can help with your CI processes.
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 13]
```diff
@@ -10,57 +10,61 @@ bannerImageAlt: 125 characters max, describes image to people unable to see it.
...
+When you're developing and deploying software, one of the first things to figure out is how to take your code and deploy your working application to a production environment where people can interact with your software.
 
-Given they can automate everything that happens between code commits and deployments, build servers are vital to CI as a concept. Let’s dig into exactly why we rate them so highly.
+Most development teams understand the importance of version control to coordinate code commits, and build servers to compile and package their software, but Continuous Integration (CI) is a big topic. Over the next few months, we’re going into detail about Continuous Integration and how two of the most popular build servers, Jenkins and GitHub Actions, can help with your CI processes.
+
+Given build servers can automate everything that happens between code commits and deployments, they're vital to CI as a concept. Let’s dig into why we rate them so highly.
```
Comment (wordlee) [inline blog/2022-01/an-introduction-to-build-servers/index.md:23]:
```suggestion

```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+
+Build servers have 3 main purposes:
+
+- Compiling committed code from your repository many times a day
+- Running automatic tests to validate code
+- Creating deployable packages and handing off to your deployment tool
+
+Without a build server you're slowed down by complicated, manual processes and the needless time constraints they introduce. For example, without a build server:
+
+- Your team will likely need to commit code before a daily deadline or during change windows.
```
Comment (TeganAli) [inline blog/2022-01/an-introduction-to-build-servers/index.md:34]:
```suggestion
- Your team will likely need to commit code before a daily deadline or during change windows
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+- Once that deadline passes, no one can commit again until someone manually creates and tests a build.
```
Comment (TeganAli) [inline blog/2022-01/an-introduction-to-build-servers/index.md:35]:
```suggestion
- After that deadline passes, no one can commit again until someone manually creates and tests a build
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+
+- Compiling committed code from your repository many times a day
+- Running automatic tests to validate code
+- Creating deployable packages and handing off to your deployment tool
+
+Without a build server you're slowed down by complicated, manual processes and the needless time constraints they introduce. For example, without a build server:
+
+- Your team will likely need to commit code before a daily deadline or during change windows
+- Once that deadline passes, no one can commit again until someone manually creates and tests a build.
+- If there are problems with code, the deadlines and manual processes delay the fixes further.
```
Comment (TeganAli) [inline blog/2022-01/an-introduction-to-build-servers/index.md:36]:
```suggestion
- If there are problems with code, the deadlines and manual processes further delay the fixes
```

File: blog/2022-01/an-introduction-to-build-servers/index.md [hunk 17]
```diff
@@ -0,0 +1,74 @@
+---
+title: An introduction to build servers and continuous integration
```
Comment (TeganAli) [inline blog/2022-01/an-introduction-to-build-servers/index.md:2]:
```suggestion
title: An introduction to build servers and Continuous Integration
```
```suggestion
title: An introduction to build servers and continuous integration
```

File: blog/2022-01/introduction-to-build-servers/index.md [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+  - CI Series
+  - Continuous Integration
+  - Jenkins
+  - GitHub Actions
+---
+
+When you're developing and deploying software, one of the first things to figure out is how to take your code and deploy your working application to a production environment where people can interact with your software.
+
+Most development teams understand the importance of version control to coordinate code commits, and build servers to compile and package their software, but Continuous Integration (CI) is a big topic. Over the next few months, we’re going into detail about Continuous Integration and how two of the most popular build servers, Jenkins and GitHub Actions, can help with your CI processes.
+
```
Comment (TeganAli) [inline blog/2022-01/introduction-to-build-servers/index.md:22]:
```suggestion
```

**Pull request merged**